bluespec maintainer flokli -> jcumming (#103487)

jcumming opened the pull request:
###### Motivation for this change

As per #103487 bluespec needs a new maintainer. I'll try a turn.

jcumming committed:
bluespec maintainer flokli -> jcumming (#103487)

flokli commented:
Something seems odd with the author email on this PR - is it your primary github email address?

jcumming committed:
jcumming -> add github and githubId

jcumming commented:
I haven't submitted many PRs lately. 

I've updated my github profile to match my commit email address, and I've updated the maintainers list to have my github account and githubId.
